src/Features/JsonPatch.SystemTextJson/build.cmd

```diff
@@ -0,0 +1,4 @@
+
+@ECHO OFF
+SET RepoRoot=%~dp0..\..\..
+%RepoRoot%\eng\build.cmd -projects %~dp0**\*.*proj %*
```

Comment:
I noticed that `startvs.cmd` points to JsonPatch.sln instead of JsonPatch.SystemTextJson.sln so you might want to update that (if you're planning on keeping the two separate).

Reply:
good catch!